modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -12,51 +12,32 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 * 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
 * 4 GB of storage
 
-== Minimum Software Requirements
+== Supported Software and Systems
 
 Verify that you use a supported version of Java and a supported operating system before you install Mule.
 
-If you use an application server or need to access a data store, verify that you use a supported application server or database.
+Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
 
-=== Supported Java Versions
+[%header,cols="15a,85a"]
+|===
+|Software |Version
+| JDK | JDK 1.8.0 (JDK 1.8.0_161 recommended), OpenJDK 8 (JDK 1.8.0_162 recommended), IBM JDK 1.8
+| OS | MacOS 10.11.x, HP-UX 11i V3, AIX 7.2, Windows 2016 Server (recommended), Windows 2012 R2 Server, Windows 10 (recommended), Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 16.04, or any other Linux distribution that runs a supported JDK and Linux kernel 3.13
```

Comment:
This should read "at least Linux Kernel 3.13", or "Linux Kernel 3.13+". Also, please update Ubuntu Server to 18.04

Reply:
Updated.